6 bugs in bubblesort.c

(and no, this code was not made by me)

main.c:62:19: error: expected ')' before numeric constant
   62 |  bubble_sort(yarr, 10);
      |                   ^~~
      |                   )
main.c:65:2: error: expected identifier or '(' before 'for'
   65 |  for (int i = 0; i < 10; i++)
      |  ^~~
main.c:65:20: error: expected '=', ',', ';', 'asm' or '__attribute__' before '<' token
   65 |  for (int i = 0; i < 10; i++)
      |                    ^
main.c:65:27: error: expected '=', ',', ';', 'asm' or '__attribute__' before '++' token
   65 |  for (int i = 0; i < 10; i++)
      |                           ^~
main.c:68:2: error: expected identifier or '(' before 'return'
   68 |  return 0;
      |  ^~~~~~
main.c:69:1: error: expected identifier or '(' before '}' token
   69 | }
      | ^

--- a/bubblesort.c
+++ b/bubblesort.c
@@ -1,16 +1,14 @@
 //NOTICE: This code has 6 bugs at the moment, so if you'd like to contribute to this repository, fix the code! Thanks.
 #include <stdio.h>
 
-void bubble_sort(extern int arr[], int len) 
+void bubble_sort(int *array, int len) 
 {
-    // arr[]: The array to sort; len: The length of the array
-	for (int i = 0; i < len; i++) {
+    for (int i = 0; i < len; i++) {
 		for (int j = 0; j < len - i - 1; j++) {
-			if (arr[j] > arr[j + 1]) {
-				// Swap arr[j] and arr[j + 1]
-				int tmp = arr[j];
-				arr[j] = arr[j + 1];
-				arr[j + 1] = tmp;
+			if (array[j] > array[j + 1]) {
+				int tmp = array[j];
+				array[j] = array[j + 1];
+				array[j + 1] = tmp;
 			}
 		}
 	}
@@ -18,37 +16,13 @@ void bubble_sort(extern int arr[], int len)
 
 int main(void) 
 {
-int yarr[] = 
-{
-		3, 
-		9, 
-		4, 
-		8, 
-		7, 
-		6,
-		1,
-		2, 
-		0,
-		10
-	};
-
-    // Sort yarr
-	bubble_sort(yarr, 10);
+	int yarr[] = {3, 9, 4, 8, 7, 6, 1, 2, 0, 10};
     
-       // Prints the elements of yarr
-	for (int i = 0; i < 10; i++)
-		printf("%d\n", yarr[i]);
-
-	return 0;
-	
-}
-	};
-
 	bubble_sort(yarr, 10);
-    
-       // Prints the elements of yarr
-	for (int i = 0; i < 10; i++)
-		printf("%d\n", yarr[i]);
 
+	printf("[");
+	for (int i = 0; i < 10; i++)
+		printf("%d, ", yarr[i]);	
+	printf("\b\b]\n");
 	return 0;
 }

--- a/bubblesort.c
+++ b/bubblesort.c
@@ -3,25 +3,25 @@
 void bubble_sort(int *array, int len) 
 {
     for (int i = 0; i < len; i++) {
-		for (int j = 0; j < len - i - 1; j++) {
-			if (array[j] > array[j + 1]) {
-				int tmp = array[j];
-				array[j] = array[j + 1];
-				array[j + 1] = tmp;
-			}
-		}
-	}
+        for (int j = 0; j < len - i - 1; j++) {
+            if (array[j] > array[j + 1]) {
+                int tmp = array[j];
+                array[j] = array[j + 1];
+                array[j + 1] = tmp;
+            }
+        }
+    }
 }
 
 int main(void) 
 {
-	int yarr[] = {3, 9, 4, 8, 7, 6, 1, 2, 0, 10};
+    int yarr[] = {3, 9, 4, 8, 7, 6, 1, 2, 0, 10};
     
-	bubble_sort(yarr, 10);
+    bubble_sort(yarr, 10);
 
-	printf("[");
-	for (int i = 0; i < 10; i++)
-		printf("%d, ", yarr[i]);	
-	printf("\b\b]\n");
-	return 0;
+    printf("[");
+    for (int i = 0; i < 10; i++)
+        printf("%d, ", yarr[i]);	
+    printf("\b\b]\n");
+    return 0;
 }

--- a/bubblesort.c
+++ b/bubblesort.c
@@ -1,4 +1,3 @@
-//NOTICE: This code has 6 bugs at the moment, so if you'd like to contribute to this repository, fix the code! Thanks.
 #include <stdio.h>
 
 void bubble_sort(int *array, int len) 

